docs/source/release/v6.9.0/SANS/Bugfixes/36115.rst

```diff
@@ -0,0 +1 @@
+- The GUI will now return to its default output options when the reduction mode is switched between 1D and 2D.
```

Comment:
Could we mention the ISIS SANS GUI here to be clear which interface the changes refer to?